app/views/reserves/_reserves_form.html.erb

```diff
@@ -5,25 +5,24 @@
       <div class="error"><%=  flash[:error] %></div>
     <%- end -%>
 	  <%- if params[:action] == "new" -%>
-  		<h2>
-  		  <%= @course.cid if @course -%>: <%= @course.title if @course -%>
-  		  <%= "(#{@course.cross_listings})" if @course and @course.cross_listings.present? %>
-  		</h2>
-  		<input type="hidden" name="reserve[cid]" value="<%= @course.cid if @course -%>" />
-  		<input type="hidden" name="reserve[desc]" value="<%= @course.title if @course -%>" />
-  		<input type="hidden" name="reserve[sid]" value="<%= @course.sid if @course -%>" />
-  		<input type="hidden" name="reserve[compound_key]" value="<%= @course.comp_key if @course -%>" />
-  		<input type="hidden" name="reserve[cross_listings]" value="<%= @course.cross_listings if @course -%>" />
-  		<div class="section"><span>Instructor:</span> <%= @course.instructor_names.join(', ') if @course -%></div>
-  		<input type="hidden" name="reserve[instructor_sunet_ids]" value="<%= @course.instructor_sunets.join(', ') if @course -%>" />
-  		<input type="hidden" name="reserve[instructor_names]" value="<%= @course.instructor_names.join(', ') if @course -%>" />
+      <h2>
+        <%= @course.cid if @course -%>: <%= @course.title if @course -%>
+        <%= "(#{@course.cross_listings})" if @course and @course.cross_listings.present? %>
+      </h2>
+      <input type="hidden" name="reserve[cid]" value="<%= @course.cid if @course -%>" />
+      <input type="hidden" name="reserve[sid]" value="<%= @course.sid if @course -%>" />
+      <input type="hidden" name="reserve[desc]" value="<%= @course.title if @course -%>" />
```

Comment:
Really just updated the tabs spacing here.  I ended up keeping the `desc` to be written the first time as a fall-back in case the course isn't in the XML (which will happen if folks try to edit courses that are not in our current feed).